Reminder-App/Reminder-App/Reminder.swift

```diff
@@ -8,13 +8,17 @@
 import Foundation
 
 struct Reminder {
+    
+    // MARK:- Instance Properties
     var title: String
     var dueDate: Date
     var notes: String? = nil
```

Comment:
I think this is a single note so variable name should also reflect it.

Reply:
Modified it.